apps/trading/client-pages/markets/use-column-defs.tsx

```diff
@@ -29,29 +29,40 @@ import { EmblemByMarket } from '@vegaprotocol/emblem';
 import { useChainId } from '@vegaprotocol/wallet-react';
 import { MarketIcon, getMarketStateTooltip } from './market-icon';
 
-const openInterestRenderer = (data: MarketMaybeWithData | undefined) => {
-  if (!data) return '-';
+const openInterestValues = (data: MarketMaybeWithData) => {
+  if (!data) return null;
   const { data: marketData, positionDecimalPlaces, decimalPlaces } = data;
-  if (!marketData) return '-';
+  if (!marketData) return null;
   const { openInterest, markPrice } = marketData;
-  const quoteName = getQuoteName(data);
+
   const openInterestPosition =
     openInterest === undefined
       ? '-'
       : addDecimalsFormatNumber(openInterest, positionDecimalPlaces);
-  const openInterestValue = toBigNum(
+  const openInterestNotional = toBigNum(
     openInterest,
     positionDecimalPlaces
   ).multipliedBy(toBigNum(markPrice, decimalPlaces));
 
-  const openInterestNotional =
+  return {
+    openInterest: openInterestPosition,
+    openInterestNotional,
+  };
+};
+
+const openInterestRenderer = (data: MarketMaybeWithData | undefined) => {
```

Comment:
You are using this as a component but not capitalizing and calling it as JSX. Something that returns JSX should be written as a component IMO

for example:

`const OpenInterestCell = () => ...`
and
`<OpenInterestCell  />`